Add more complete Sass requirements to the README (#4638)
uswds/uswds: README.md

#### Sass compilation requirements
**Sass Module syntax:** USWDS 3.0 requires a modern Sass compiler that can parse Sass Module syntax. 

**Proper load paths:** Using USWDS 3.0 and packages requires compiling your Sass with load paths. Load paths tell your Sass compiler where to look for USWDS packages. Any compiler needs to include the following load paths:

```js
"./node_modules/@uswds",
"./node_modules/@uswds/uswds/packages"
```

In a gulpfile, use `includePaths` in your `sass()` function.
```js
  sass({
    includePaths: [
      "./node_modules/@uswds",
      "./node_modules/@uswds/uswds/packages",
    ],
    outputStyle: "expanded"
  })
```

In webpack, include `includePaths` within the sassOptions of your Sass loader:

```js
loader: "sass-loader",
options: {
  sourceMap: true,
  sassOptions: {
    includePaths: [
      "./node_modules/@uswds",
      "./node_modules/@uswds/uswds/packages",
    ],
  },
},
```

> **Autoprefixing:** The design system requires autoprefixing to work properly. Autoprefixing uses a service like [gulp-autoprefixer](https://github.com/sindresorhus/gulp-autoprefixer) to automatically add vendor prefixes to the precompiled stylesheets. Don't add vendor prefixes to your custom styles manually — it is more reliable to use autoprefixing. We use the following autoprefixer settings via `.browserslistrc` config:

Reviewer: ```suggestion
**Autoprefixing:** The design system requires autoprefixing to work properly. Autoprefixing services like [gulp-autoprefixer](https://github.com/sindresorhus/gulp-autoprefixer) automatically add vendor prefixes to CSS rules. Don't add vendor prefixes to your custom styles manually — it is more reliable to use autoprefixing. We use the following autoprefixer settings via `.browserslistrc` config:
```

not dead
```

> **Minfiers and sourcemaps:** We recommend using a minifier like [csso](https://github.com/css/csso) to compress your final compiled CSS and sourcemaps like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.

Reviewer: Typo in the word 'minifiers'. Additionally, could we break minifiers and sourcemaps into their own sections, or maybe provide a pause in the sentence to separate the thoughts? I read this first as "compress your final compiled CSS and sourcemaps". 
```suggestion
**Minifiers and sourcemaps:** We recommend using a minifier like [csso](https://github.com/css/csso) to compress your final compiled CSS and sourcemaps like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.
```

> Using [`uswds-compile`](https://github.com/uswds/uswds-compile) includes all these requirements by default and can be a good way to build a reliable compile workflow into your project.

Reviewer: On first read, it was a little confusing to me if the requirements referenced are for minifier/sourcemaps or all Sass compilation. I also wonder if it would be helpful to move this note about `uswds-compile` to the top of the Sass requirements section.

One idea:
```suggestion
Using [`uswds-compile`](https://github.com/uswds/uswds-compile) includes all these Sass compilation requirements by default and can be a good way to build a reliable compile workflow into your project.
```
